server/public/index.html

```diff
@@ -245,6 +252,91 @@
   .audienceCard_fWZI .button--outline:hover {
     background-color: rgba(26, 54, 180, 0.1) !important;
   }
+
+  /* Stack hero points vertically */
+  .point__uAb {
```

Comment:
thoughts on this being here vs the overall design system css so we can reuse elsewhere?